Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM alpine:3.5
+ARG arch=amd64
+FROM multiarch/alpine:${arch}-latest-stable
```

Comment:
Please change to use a specific version ie `${arch)-v3.5`, best practice is to never use `latest` in production

Reply:
If you could test on `v3.8` and the bump, that would be appreciated.